sdk/deepbook-v3/test/client.test.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import path from 'path';
+import { beforeAll, describe, expect, test } from 'vitest';
+
+import { DeepBookClient, DeepBookConfig } from '../src';
+import { CoinMap } from '../src/utils/constants';
+import { publishPackage, setupSuiClient, TestToolbox } from './setup';
+import { writeToml } from './helper/toml';
+
+let toolbox!: TestToolbox;
+let coins: CoinMap;
+let deepbookPackageId: string;
+let deepbookRegistryId: string;
+let deepbookAdminCap: string;
+
+beforeAll(async () => {
+	toolbox = await setupSuiClient();
+	const tokenSourcesPath = path.join(__dirname, 'data/token');
```

Comment:
I'd probably wrap all this up as a method on toolbox so you don't have to repeat it in multiple test files